tests/test_integration.py

```diff
@@ -0,0 +1,91 @@
+"""Tests to check the full era5cli workflow."""
+
+import pytest
+
+import era5cli.cli as cli
```

Comment:
Can import be made simpler by using
```suggestion
from era5cli.cli import main
```
would also need renaming on line 89 from `cli.main` to `main`